docker/env_file_integrations_template

```diff
@@ -3,4 +3,6 @@
 ### IMPORTANT: don't change these values unless you know what you are doing. It can have breaking changes!
 
 # Applies to all integrations, choose from: INFO (recommended), ERROR, DEBUG.
-LOG_LEVEL=INFO
\ No newline at end of file
+LOG_LEVEL=INFO
+# For phoneinfoga analyzer 
+# NUMVERIFY_API_KEY=
```

Comment:
isn't there a way to leverage classic Analyzer Secrets for this API key? In that way, every user can configure it and use its own API key.
I don't like to have it configured here because this API key would be shared among all the users. This could be an unwanted behavior.

Reply:
hi there! I took a look at their code base. The problem at hand involves the extraction of all the API keys from the from the `integration .env` file instead of directly using the key as authorization header when making calls. This is hard coded in their code base. 
One work around could be making a new container everytime a new user selects the analyzer. This would have some infra implications but would allow us to have unique containers of the analyzer with its own env file for each individual user. (not a good approach tho)
I have raised an (issue)[https://github.com/sundowndev/phoneinfoga/issues/1393] for further enquiry.
In the meantime we could update the analyzer configs to take API keys from the user in the interface and implement that logic whenever they solve our problem, till then we might have to leverage `integration env` only.

Reply:
thank you for the efforts. I would like to avoid to create complex things for this problem, I like to keep things simple if possible.
I suggest waiting for an answer from them. In case they do not answer within a week, we consider the problem unsolved and so we'll merge your actual PR even if there is that problem. In that case, we would need to specify more information about it in the docs, just to be explicit about this problem with the users.

Reply:
for sure! In the meantime can you guide me to another issue i can take up?

Reply:
this is a sightly different analyzer: https://github.com/intelowlproject/IntelOwl/issues/1103.
This is a really important source for this project